src/ansys/fluent/core/solver/flobject.py

```diff
@@ -359,6 +359,36 @@ class BooleanList(SettingsBase[BoolListType], Property):
     _state_type = BoolListType
 
 
+def _command_query_name_filter(self, command: bool = True, prefix: str = "") -> list:
+    """Auto completer info of commands and queries."""
+    ret = []
+    if command:
+        command_or_query_names = self.command_names
+    else:
+        command_or_query_names = self.query_names
+    for command_or_query_name in command_or_query_names:
+        if command_or_query_name.startswith(prefix):
+            command_or_query_name = getattr(self, command_or_query_name)
+            if command_or_query_name.is_active():
+                if command:
+                    ret.append(
+                        [
+                            command_or_query_name,
+                            Command.__name__,
+                            command_or_query_name.__doc__,
+                        ]
+                    )
+                else:
+                    ret.append(
+                        [
+                            command_or_query_name,
+                            Query.__name__,
+                            command_or_query_name.__doc__,
+                        ]
+                    )
```

Comment:
@hpohekar 
- it's only the selection between `Command` and `Query` that depends on the value of `command`
- `command` could be renamed `is_command` or similar because `command` reads as if it is a command object
- `command_or_query_names` and `command_or_query_name` are a bit long-winded. `name` is fine for the latter

Reply:
Done. Thank you.